lib/Common/Core/FaultInjection.cpp

```diff
@@ -1398,11 +1405,22 @@ namespace Js
             return EXCEPTION_CONTINUE_EXECUTION;
         }
 
-        struct AutoValue {
-            AutoValue() { inExceptionHandler = true; }
-            ~AutoValue() { inExceptionHandler = false; }
-        } autoVal;
-        FaultInjection::Global.FaultInjectionAnalyzeException(ExceptionInfo);
+        inExceptionHandler = true;
+
+        __try 
```

Comment:
The reason I asked because of the cross platform work. This will break the build.